lib/repl.js

```diff
@@ -115,6 +115,11 @@ function REPLServer(prompt,
   // just for backwards compat, see github.com/joyent/node/pull/7127
   self.rli = this;
 
+  const savedRegExMatches = ['', '', '', '', '', '', '', '', '', ''];
+  const regExMatcher = new RegExp('^\u0000(.*)\u0000(.*)\u0000(.*)\u0000(.*)' +
+                                  '\u0000(.*)\u0000(.*)\u0000(.*)\u0000(.*)' +
+                                  '\u0000(.*)$');
```

Comment:
Could this be just `\u0000(.*)`? I'm trying to understand how tis actually works.